doc/ale-go.txt

```diff
@@ -50,4 +50,14 @@ g:ale_go_gometalinter_options                   *g:ale_go_gometalinter_options*
 
 
 ===============================================================================
+gofmt                                                            *ale-go-gofmt*
```

Comment:
Check out the Travis CI results. It's complaining because there's no table of contents entry which references this. You should add `gofmt` to the table of contents.